lib/datadog/statsd.rb

```diff
@@ -93,16 +93,19 @@ def self.VERSION
     # @param [Integer] port your statsd port
     # @option opts [String] :namespace set a namespace to be prepended to every metric name
     # @option opts [Array<String>] :tags tags to be added to every metric
-    # @option opts [Integer] :max_buffer_size max messages to buffer
+    # @option opts [Integer] :max_buffer_bytes max messages to buffer
```

Comment:
this option was never released, so it should be fine to change it
users that used the additional argument before will get a "ArgumentError" and then have to read the docs on how to upgrade